hack/make/.build-deb/rules

```diff
@@ -22,6 +22,10 @@ override_dh_strip:
 override_dh_auto_install:
 	mkdir -p debian/docker-engine/usr/bin
 	cp -aT "$$(readlink -f bundles/$(VERSION)/dynbinary/docker)" debian/docker-engine/usr/bin/docker
+	cp -aT /usr/local/bin/containerd debian/docker-engine/usr/bin/docker-containerd
+	cp -aT /usr/local/bin/containerd-shim debian/docker-engine/usr/bin/docker-containerd-shim
+	cp -aT /usr/local/bin/containerd-ctr debian/docker-engine/usr/bin/docker-ctr
+	cp -aT /usr/local/bin/containerd-runc debian/docker-engine/usr/bin/docker-runc
```

Comment:
these are using the new names with docker- prepended. This will require the binary renames before this will work correctly.

See: https://github.com/docker/containerd/pull/146